test/integration/hooks.spec.js

```diff
@@ -49,4 +50,33 @@ describe('hooks', function() {
       done();
     });
   });
+
+  it('can fail a test in an afterEach hook', function(done) {
```

Comment:
wouldn't it be easier to just look at the output of `runMochaJSON()`?

Reply:
I changed to use it.